internal/ledger/store/db_mock_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"sort"
```

Comment:
please don't call files with `_mock_` it's could be misleading for tools and developers

Reply:
Reached an agreement that don't touch it, since we will remove it soon